### DISCUSSION TITLE
GDAL is outdated (3.6.4; current is 3.7.1.1)

### DISCUSSION THREAD

**Discussion by paulschreiber** [Homebrew/discussions#4666] [Everyday usage]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.1.0
ORIGIN: https://github.com/Homebrew/brew
HEAD: ac93842f8d13913f462aedb246cfacd08d6b7f05
Last commit: 5 days ago
Core tap JSON: 24 Jul 18:52 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_EDITOR: vi
HOMEBREW_MAKE_JOBS: 10
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: 10-core 64-bit arm_firestorm_icestorm
Clang: 14.0.3 build 1403
Git: 2.41.0 => /opt/homebrew/bin/git
Curl: 7.88.1 => /usr/bin/curl
macOS: 13.4.1-arm64
CLT: 14.3.1.0.1.1683849156
Xcode: 14.3.1
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

GDAL is outdated. Homebrew installs 3.6.4; current is 3.7.1.1.
```
$ pip list --outdated
Package Version Latest  Type
------- ------- ------- -----
GDAL    3.6.4   3.7.1.1 sdist
```

Should I file a bug against `homebrew-core`? Or is there a better method to request this be updated?

**Comment by osalbahr** [comment]
Try `brew livecheck gdal` and `brew bump gdal`. If either show that it is outdated, I recommend checking [`brew help bump-formula-pr`](https://docs.brew.sh/Manpage#bump-formula-pr-options-formula) for a quick way to open a PR to bump the version.

Edit: looks like there’s already a version bump PR (which `brew bump` should indicate):
- https://github.com/Homebrew/homebrew-core/pull/134485
